doc/src/odin/top.md

```diff
@@ -5,5 +5,7 @@ Odin II is used for logic synthesis and elaboration, converting a subset
 of the Verilog Hardware Description Language (HDL) into a BLIF netlist.
 
 [Install/Quickstart](./install.md) 
-[User Guide](./user_guide.md)
-[Developper Guide](./dev_guide.md)
\ No newline at end of file
+
+[User Guide](user_guide/user_guide.md)
```

Comment:
I would add back the `./` in the paths

Reply:
?

Reply:
This links the pages together

Reply:
it uses path to do so, and in most your path will look like `./user_guide/user_guide.md` but here you do `user_guide/user_guide.md`. 
I just wanted it to be consistent

Reply:
Ou okay, I thought the ./ format was just when it's in the same folder